subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/IndyDeployment.java

```diff
@@ -203,7 +203,7 @@ public DeploymentInfo getDeployment( final String contextRoot )
 
                                                       .addFilter( slashTolerationFilter )
                                                       .addFilterUrlMapping( slashTolerationFilter.getName(),
-                                                                            "//*", DispatcherType.REQUEST )
+                                                                            "/api/*", DispatcherType.REQUEST )
```

Comment:
Hi, I suggest we filter "*" here instead of "/api/*" then make it possible to tolerate the slashes everywhere in path.
Some issues raised before like : //api/.., /api/..//.. Related PR merged in master: https://github.com/Commonjava/indy/pull/1903

Reply:
Sure, thanks for the info. :)

Reply:
I did cherry-pick from #1903 , thanks.

Reply:
LGTM, thanks!